tests/integration/integration_test.go

```diff
@@ -699,10 +699,10 @@ status: {}
 
 			By("creating the artifacts")
 			Expect(files).To(ContainElements(
-				"artifacts/bitnami-nginx/ConfigMap.yaml",
-				"profile-installation.yaml",
-				"artifacts/bitnami-nginx/HelmRelease.yaml",
-				"artifacts/bitnami-nginx/HelmRepository.yaml",
+				"nginx/artifacts/bitnami-nginx/ConfigMap.yaml",
```

Comment:
didn't I already do this?

Reply:
🤷‍♀️  says my branch is up to date.

it was failing for other reasons tho, this was just extra

Reply:
https://github.com/weaveworks/pctl/commit/08f02c34024343fa3a9fc8770981af566399e7c0#diff-e4ede531ad8643dc7af3ffb5ddf644bdfa5c2f8a45430f580f0fa687ffab06efR702

What the heck. :D

Reply:
wonky diff url... but this is the PR: https://github.com/weaveworks/pctl/pull/142/files#

Reply:
Oh!!!!!!!! It was your merge conflict resolution which reverted my fix. :D

Reply:
https://github.com/weaveworks/pctl/commit/a139fa55ef6501a159a75438eb7e1397e5bd0151#diff-e4ede531ad8643dc7af3ffb5ddf644bdfa5c2f8a45430f580f0fa687ffab06efL702